### PR TITLE
new CTDMO cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**

### CODE REVIEW

File: calibration/CTDMOR/CGINS-CTDMOR-13654__20200223.csv [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+37-13654,CC_ptca0,5.241448e+05,
+37-13654,CC_ptca1,-1.403068e-02,
+37-13654,CC_ptca2,1.021928e-02,
+37-13654,CC_ptcb0,2.520675e+01,
+37-13654,CC_ptcb1,-6.500000e-04,
+37-13654,CC_ptcb2,0.000000e+00,
+37-13654,CC_ptempa0,2.104110e+02,
+37-13654,CC_ptempa1,-1.051269e-01,
+37-13654,CC_ptempa2,4.041920e-06,
+37-13654,CC_wbotc,0.000000e+00,
```
Comment (dbogorff) [inline calibration/CTDMOR/CGINS-CTDMOR-13654__20200223.csv]: Looks good!

File: calibration/CTDMOG/CGINS-CTDMOG-13430__20200226.csv [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+37-13430,CC_ptca0,5.241581e+05,
+37-13430,CC_ptca1,2.659016e+00,
+37-13430,CC_ptca2,1.692148e-01,
+37-13430,CC_ptcb0,2.512863e+01,
+37-13430,CC_ptcb1,-2.743142e-04,
+37-13430,CC_ptcb2,0.000000e+00,
+37-13430,CC_ptempa0,-6.640089e+01,
+37-13430,CC_ptempa1,5.337702e-02,
+37-13430,CC_ptempa2,-9.967495e-07,
+37-13430,CC_wbotc,1.531297e-07,
```
[the remark this answers is not in the document]
Comment (batryn) [reply calibration/CTDMOG/CGINS-CTDMOG-13430__20200226.csv]: Looks good

File: calibration/CTDMOG/CGINS-CTDMOG-13593__20200214.csv [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+37-13593,CC_ptca0,5.251193e+05,
+37-13593,CC_ptca1,-5.235302e+00,
+37-13593,CC_ptca2,3.398475e-01,
+37-13593,CC_ptcb0,2.520712e+01,
+37-13593,CC_ptcb1,2.250000e-04,
+37-13593,CC_ptcb2,0.000000e+00,
+37-13593,CC_ptempa0,1.447411e+02,
+37-13593,CC_ptempa1,-6.206082e-02,
+37-13593,CC_ptempa2,-1.051346e-06,
+37-13593,CC_wbotc,6.487438e-07,
```
[the remark this answers is not in the document]
Comment (batryn) [reply calibration/CTDMOG/CGINS-CTDMOG-13593__20200214.csv]: Looks good

**Comment by snwhite** [comment]
[Instrument Cal CSVs [DT-69]](https://trello.com/c/CbhVKyjb/69-instrument-cal-csvs-dt-69)

**Pull request merged**